python/doc/examples/reliability_sensitivity/reliability/plot_nais.py

```diff
@@ -0,0 +1,232 @@
+"""
+Non parametric Adaptive Importance Sampling (NAIS)
+==================================================
+"""
+# %%
+#
+# The objective is to evaluate a probability from the Non parametric Adaptive Importance Sampling (NAIS) technique.
+#
+# We consider the four-branch function :math:`g : \mathbb{R}^2 \rightarrow \mathbb{R}` defined by:
+#
+# .. math::
+#   \begin{align*}
+#   g(X) = \min \begin{pmatrix}5+0.1(x_1-x_2)^2-\frac{(x_1+x_2)}{\sqrt{2}}\\
```

Comment:
Here and at other places there are mentions of "g(X)". Try to search for them in order to replace them with "g(\vect{X})". After that the PR will be good to go IMO.